components/site-search/element.js

```diff
@@ -361,7 +361,11 @@ export class MDNSiteSearch extends L10nMixin(LitElement) {
                         <p class="site-search-results__description">
                           ${result.highlight.body &&
                           result.highlight.body.length > 0
-                            ? result.highlight.body.map((b) => unsafeHTML(b))
+                            ? result.highlight.body.flatMap((b, i) =>
+                                i === 0
+                                  ? [unsafeHTML(b)]
+                                  : [html` … `, unsafeHTML(b)],
+                              )
```

Comment:
I think it would be cleaner to use Lit's [`join` directive](https://lit.dev/docs/templates/directives/#join):

```suggestion
                            ? join(result.highlight.body.map((b) => unsafeHTML(b)), html` … `),
                              )
```